plugins/procedures/hooks/query-shape-guard.sh

```
@@ -123,7 +121,7 @@ SCAN="${SCAN//>\/dev\/null/ }"
 
 # Constructs that hide a segment from this scan deny outright.
 case "$SCAN" in
-    *'$('*|*'`'*|*'<('*|*$'\n'*) qsg_deny "QUERY-SHAPE-GUARD: command substitution hides a segment from the shape check — run the digest read, the pipeline, and the justfile probe as plain segments." ;;
+    *'$('*|*'`'*|*'<('*|*$'\n'*) qsg_deny "QUERY-SHAPE-GUARD: command substitution hides a segment from the shape check — run the pipeline and the justfile probe as plain segments." ;;
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)